tests/unit/worker_manager/slurm_batch_worker_manager_test.py

```diff
@@ -15,6 +15,7 @@ def test_base_command(self):
             partition='some_partition',
             worker_executable='cl-worker',
             worker_idle_seconds='888',
+            worker_download_dependencies_max_retries=5,
```

Comment:
Why 5 here instead of 3?

Reply:
Just for to test if setting a value other than the default value of 3 gets respected.